server/src/main/java/io/druid/server/initialization/BatchDataSegmentAnnouncerConfig.java

```diff
@@ -37,6 +37,18 @@
   @Min(1024)
   private long maxBytesPerNode = 512 * 1024;
 
+  // Skip LoadSpec from segment announcements
+  @JsonProperty
+  private boolean skipLoadSpec = false;
```

Comment:
coordinator has end point for getting info for a particular segmentId which includes loadSpec too.